sql/src/test/resources/calcite/tests/window/wikipediaSimplePartition.sqlTest

```diff
@@ -6,29 +6,31 @@ sql: |
     FLOOR(__time TO HOUR) t,
     SUM(delta) delta,
     SUM(SUM(delta)) OVER (PARTITION BY countryIsoCode) totalDelta,
-    LAG(FLOOR(__time TO HOUR),  2) OVER (PARTITION BY countryIsoCode) laggardTime,
-    LEAD(FLOOR(__time TO HOUR),  1) OVER (PARTITION BY countryIsoCode) leadTime,
-    FIRST_VALUE(SUM(delta)) OVER (PARTITION BY countryIsoCode) AS firstDelay,
-    LAST_VALUE(SUM(delta)) OVER (PARTITION BY countryIsoCode) AS lastDelay,
-    NTILE(3) OVER (PARTITION BY countryIsoCode) AS delayNTile
+    LAG(FLOOR(__time TO HOUR),  2) OVER (PARTITION BY countryIsoCode ORDER BY FLOOR(__time TO HOUR)) laggardTime,
+    LEAD(FLOOR(__time TO HOUR),  1) OVER (PARTITION BY countryIsoCode ORDER BY FLOOR(__time TO HOUR)) leadTime,
+    FIRST_VALUE(SUM(delta)) OVER (PARTITION BY countryIsoCode ORDER BY FLOOR(__time TO HOUR)) AS firstDelay,
+    LAST_VALUE(SUM(delta)) OVER (PARTITION BY countryIsoCode ORDER BY FLOOR(__time TO HOUR)) AS lastDelay,
+    NTILE(3) OVER (PARTITION BY countryIsoCode ORDER BY FLOOR(__time TO HOUR)) AS delayNTile
```

Comment:
why change test query instead of adding new one?

Reply:
On the new Calcite, these queries needed to change, Gian/Eric introduced them. I'll ask